test/cmocka/src/audio/buffer/buffer_wrap.c

```diff
@@ -33,7 +33,13 @@ static void test_audio_buffer_write_fill_10_bytes_and_write_5(void **state)
 
 	uint8_t bytes[10] = {0, 1, 2, 3, 4, 5, 6, 7, 8, 9};
 
-	memcpy_s(buf->stream.w_ptr, test_buf_desc.size, &bytes, 10);
+	int i;
+	uint8_t *ptr;
+
+	for (i = 0; i < 10; i++) {
+		ptr = audio_stream_write_frag(&buf->stream, i, sizeof(uint8_t));
```

Comment:
Please don't use magic numbers. Just create a macro with a meaningful name. 

e.g

```
#define BUF_MAX_SIZE 16
```

Reply:
It should be OK to use ARRAY_SIZE inside for() and then just change the test input.

Reply:
changed